doc/attr.xml

```diff
@@ -1439,6 +1439,42 @@ gap> ChromaticNumber(D);
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="ChromaticNumberFunc">
+<ManSection>
+    <Func Name="ChromaticNumber" Arg="digraph, colouring_algorithm" 
+        Label="for a digraph and colouring algorithm"/>
+    <Returns> A non-negative integer.</Returns>
+    <Description>
+      Alterative implementation of the Chromatic Number attribute
+      (See <Ref Attr="ChromaticNumber"/>) that allows the specific algorithm 
+      used to be selected. 
+
+      If <A>digraph</A> is a digraph without loops (see <Ref
+      Prop="DigraphHasLoops"/>), then <C>ChromaticNumber</C> uses
+      <A>colouring algorithm</A> (which is given as a globally bound object)
+      to find the least non-negative integer <C>n</C> such that there is a
+      proper colouring of <A>digraph</A> with <C>n</C> colours. See
+      <Ref Attr="ChromaticNumber"/> for the definition of a proper colouring.
+
+      The current algorithm implementations are:
+      <List>
+          <Item><A>DigraphColouringAlgorithmLawler</A> - Lawler's Algorithm 
+          <Cite Key="Law1976"/></Item>
+          <Item><A>DigraphColouringAlgorithmByskov</A> - Byskov's Algorithm 
```

Comment:
`<A>` tags are reserved for the specific names of arguments. So `<C>` tags are more appropriate here.
```suggestion
          <Item><C>DigraphColouringAlgorithmLawler</C> - Lawler's Algorithm 
          <Cite Key="Law1976"/></Item>
          <Item><C>DigraphColouringAlgorithmByskov</C> - Byskov's Algorithm 
```